### DISCUSSION TITLE
popper.js.org offline?

### DISCUSSION THREAD

**Discussion by rajatrocks** [floating-ui/floating-ui#2797] [Q&A] [answered]
Hi there - the [documentation website](https://popper.js.org) seems to be offline as of this morning with a 302 error. Has anyone else noticed this?

thanks, - rajat

**Comment by FezVrasta** [comment, marked as answer]
Docs are still available here https://popper.js.org/docs/v2/

The home page now redirects to floating ui

**Reply by rajatrocks** [reply]
Thanks! I think there was a temporary glitch where js.org was having a problem.

**Comment by rajniszp** [comment]
Well, no longer
